test/k8s/manifests/sctp/iperf3.yaml

```diff
@@ -0,0 +1,101 @@
+# MIT License
+
+# Copyright (c) 2018 Pharb
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: iperf3-server-deployment
+  labels:
+    app: iperf3-server
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: iperf3-server
+  template:
+    metadata:
+      labels:
+        app: iperf3-server
+    spec:
+      affinity:
+        nodeAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 1
+            preference:
+              matchExpressions:
+              - key: kubernetes.io/role
+                operator: In
+                values:
+                - master
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          operator: Exists
+          effect: NoSchedule
+      containers:
+      - name: iperf3-server
+        image: networkstatic/iperf3
```

Comment:
It looks like the full image path for this is `docker.io/networkstatic/iperf3`, which unfortunately is on dockerhub, hence can hit issues with dockerhub ratelimiting.

This (and other image references) should at least contain the full path + version string + sha of the image, so that we can manage any potential (breaking) version changes from the upstream docker image maintainer.

Furthermore, we should pre-pull the image into the VM image as well to avoid issues where the CI fails because of dockerhub ratelimiting. For that, it needs to be integrated here: https://github.com/cilium/packer-ci-build/blob/master/provision/pull-images.sh . Maybe we can work together on Slack to get the VM image bits updated correctly to avoid this issue.

Reply:
I see. I asked about this on Slack and was told it was fine: https://cilium.slack.com/archives/C7PE7V806/p1655397271386549.

It's not a super complicated image (just ubuntu + iperf3), so we can create it anywhere. I'll sync up with you tomorrow then to find a path forward. Thanks for walking me through all of this! 😄

Reply:
:+1: yeah I think the image itself should be fine, there's just two remaining nits: (1), the name should be fully specified so that we don't randomly break CI because upstream decides to push a new image that's somehow incompatible/broken/etc and (2) the image should be cached so that we don't break CI because of dockerhub ratelimiting.

Reply:
https://github.com/cilium/packer-ci-build/pull/339

Reply:
Also updated the test to use a fully qualified name with hash.

Reply:
Now merged: #20596